pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotHelixTaskResourceManager.java

```diff
@@ -258,6 +262,50 @@ public synchronized String submitTask(List<PinotTaskConfig> pinotTaskConfigs, St
     return taskStates;
   }
 
+  /**
+   * This method returns a count of sub-tasks in various states, given the top-level task name.
+   * @param parentTaskName (e.g. "Task_TestTask_1624403781879")
+   * @return TaskCount object
+   */
+  public synchronized TaskCount getTaskCount(String parentTaskName) {
+    JobContext jobContext = _taskDriver.getJobContext(getHelixJobName(parentTaskName));
```

Comment:
JobContext could be null according to this code from Helix:
```
 protected static JobContext getJobContext(HelixPropertyStore<ZNRecord> propertyStore, String jobResource) {
    ZNRecord r = (ZNRecord)propertyStore.get(Joiner.on("/").join("/TaskRebalancer", jobResource, new Object[]{"Context"}), (Stat)null, AccessOption.PERSISTENT);
    return r != null ? new JobContext(r) : null;
  }
 ```
 It'd be good to handle the null case here.

Reply:
Will do.